### PR TITLE
New bluecross icons and rework to thunderlord

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Bluecross - Handmade pistol (donst spawn)
Sant and Mana Form heaven now have their own uniquic icon states

Reworks and resprites thunderload to Kings Shotgun to be a bit more uniquic and fancy

### CODE REVIEW

**Comment by cdb-is-not-good** [comment]
few spelling errors but looks mostly good.

**Review by cdb-is-not-good — CHANGES_REQUESTED** [review]
Spellcheck

File: code/modules/projectiles/guns/oddity_items.dm [hunk 1]
```diff
@@ -5,20 +5,24 @@
 	name = "\"Finger of God\" handmade pistol"
 	desc = "An anomalous weapon created by an unknown person (or group?), their work marked by a blue cross, these weapons are known to vanish and reappear when left alone. \
 	Whoever made this particular model did exceptional work. Unlike lesser handmade pistols, this one jams less and has much better shot capability. This one chambers .60-06 anti material ammo."
+	icon = 'icons/obj/guns/projectile/hm_bluecross.dmi'
 	jam_chance = 5
 	damage_multiplier = 1.36
 	recoil_buildup = 25
 	max_shells = 6
 	price_tag = 750
+	matter = list(MATERIAL_PLASTEEL = 8, MATERIAL_WOOD = 6)
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/oddity_items.dm:14]:
```suggestion
	matter = list(MATERIAL_PLASTEEL = 8, MATERIAL_WOOD = 6) //Its now made of plasteel to show its whatever
```

File: code/modules/projectiles/guns/oddity_items.dm [hunk 2]
```diff
@@ -5,20 +5,24 @@
...
 
 /obj/item/gun/energy/sniperrifle/saint
 	name = "\"Saint\" laser rifle"
 	desc = "An anomalous weapon created by an unknown person (or group?), their work marked by a blue cross, these weapons are known to vanish and reappear when left alone. \
-	Unlike a standard valkyrie this rifle has an over charged coolant mechanism and redundant power loop to conserve energy between shots."
+	The normal \"Valkyrie\" design meeting the improvements with Lightfall design in a wooden frame with a golden barrel to help conduct the extra little bit of juice to hit as hard as possible."
 	charge_cost = 200
 	fire_delay = 25
 	price_tag = 3750
+	matter = list(MATERIAL_PLASTEEL = 20, MATERIAL_WOOD = 8, MATERIAL_GOLD = 9, MATERIAL_URANIUM = 6)
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/oddity_items.dm:24]:
```suggestion
	matter = list(MATERIAL_PLASTEEL = 20, MATERIAL_WOOD = 8, MATERIAL_GOLD = 9, MATERIAL_URANIUM = 6) //The normal valk needs silver this is gold plated!
```

**Review by Trilbyspaceclone — COMMENTED** [review]
Mew

**Review by cdb-is-not-good — APPROVED** [review]

**Pull request merged**